AGENT.md

```diff
@@ -0,0 +1,105 @@
+# Agent Instructions for Vertex SDK
+
+This file provides guidance to LLMs when working with code in this repository.
+
+## Repository Overview
+
+The Vertex TypeScript SDK is a monorepo containing utilities for interacting with the Vertex Protocol API and contracts. The project uses Lerna for workspace management and provides a comprehensive SDK for trading on Vertex.
+
+## Key Commands
+
+### Development
+
+- `yarn build` - Build all packages in the monorepo using Lerna
+- `yarn clean` - Clean all packages
+- `yarn dev` - Run development mode for all packages
+- `yarn test` - Run Jest tests across the entire codebase
+- `yarn lint` - Run ESLint with auto-fix and Prettier formatting
+- `yarn typecheck` - Run TypeScript type checking for all packages
+- `yarn gen-typedoc` - Generate TypeDoc documentation for all packages
+
+### Testing
+
+- E2E tests in `apps/e2e`
+- `yarn --cwd apps/e2e e2e` - Run all E2E tests
+- `yarn --cwd apps/e2e e2e:client` - Run client-specific E2E tests
+- `yarn --cwd apps/e2e e2e:engine` - Run engine-client E2E tests  
+- `yarn --cwd apps/e2e e2e:indexer` - Run indexer-client E2E tests
+- `yarn --cwd apps/e2e e2e:trigger` - Run trigger-client E2E tests
+
+### Package Management
+
+- `yarn link-local` / `yarn unlink-local` - Link/unlink packages for local development
+- `yarn publish-all` - Clean, build, and publish all packages via Lerna
+- `yarn depcruise:all` - Analyze package dependencies and detect circular dependencies
+
+### Individual Package Scripts
+
+Each package in `packages/` has these common scripts:
+- `yarn build` - Build the specific package
+- `yarn clean` - Clean build artifacts  
+- `yarn dev` - Watch mode for development
+- `yarn lint` - Check linting rules only
+- `yarn lint:fix` - Fix linting issues automatically
+- `yarn typecheck` - Type check without emitting files
+
+## Architecture
+
+### Monorepo Structure
+
+The project follows a monorepo pattern with these core packages:
+
+1. **`@vertex-protocol/client`** - Main entry point that composes all other packages into a unified `VertexClient`
+2. **`@vertex-protocol/engine-client`** - Handles off-chain matching engine communication
+3. **`@vertex-protocol/indexer-client`** - Provides indexer queries for historical data
+4. **`@vertex-protocol/trigger-client`** - Manages trigger service for stop/TP/SL orders
```

Comment:
There isn't any notion of TP/SL orders in the SDK

Reply:
There is mention in README.md. I guess this is where it picked up. I guess just add `for stop orders`?

Reply:
Ahh yeah 👍